fonts/generate.py

```diff
@@ -148,13 +148,9 @@ def extract_fonts(opts: Namespace) -> bool:
         log.error("Could not read %s. Does it exist?", font_pth)
         return False
 
-    if not os.access(output_pth, os.W_OK):
```

Comment:
I think it's actually supposed to be checking if `data_pth` exists. So it's incorrect to check `output_pth`, but we still should check if `data_pth` exists (which I think we don't do... )

Reply:
Then `os.W_OK` is also wrong?

Reply:
We need to be able to write to the `data_pth`, don't we?